### PR TITLE
[Patch] Updated copyright and added default value

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

File: lib/src/footer/Footer.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import FooterPropsType from "./types";
 const DxcFooter = ({
   socialLinks,
   bottomLinks,
-  copyright = "",
+  copyright = "© DXC Technology 2022. All rights reserved.",
```
Comment (GomezIvann) [inline lib/src/footer/Footer.tsx:13]: We keep having the same problem as before with this approach (update every year the copyright message). My proposal: 

Sin título

And as the default value for the prop don't give anyone.

File: docs/src/pages/overview/Overview.jsx [hunk 2]
```diff
@@ -368,7 +368,7 @@ function Overview() {
             { text: "Facebook", href: "http://www.google.com" },
             { text: "Instagram", href: "http://www.google.com" },
           ]}
-          copyright="© DXC Technology 2020. All rights reserved."
+          copyright="© DXC Technology 2022. All rights reserved."
```
Comment (GomezIvann) [inline docs/src/pages/overview/Overview.jsx:371]: Remove this line. This Footer (and every other one in our CDK must use the preset value).

File: docs/src/pages/components/Components.jsx [hunk 3]
```diff
@@ -156,7 +156,7 @@ function Components() {
             { text: "Facebook", href: "http://www.google.com" },
             { text: "Instagram", href: "http://www.google.com" },
           ]}
-          copyright="© DXC Technology 2020. All rights reserved."
+          copyright="© DXC Technology 2022. All rights reserved."
```
Comment (GomezIvann) [inline docs/src/pages/components/Components.jsx:159]: Remove this line. This Footer (and every other one in our CDK must use the preset value).

**Review by GomezIvann — CHANGES_REQUESTED** [review]

**Comment by GomezIvann** [comment]
Adding to the review one more thing:

- Remove every `copyright` appearance in the CDK docs, it must now use the preset value.

image

File: lib/src/footer/Footer.tsx (hunk 1 above)
Comment (Lisa159) [reply lib/src/footer/Footer.tsx]: Done

File: docs/src/pages/components/Components.jsx (hunk 3 above)
Comment (Lisa159) [reply docs/src/pages/components/Components.jsx]: Done

File: docs/src/pages/overview/Overview.jsx (hunk 2 above)
Comment (Lisa159) [reply docs/src/pages/overview/Overview.jsx]: Done

**Review by GomezIvann — APPROVED** [review]

**Pull request merged**